packages/lang/useB3Lang.ts

```diff
@@ -1,26 +1,22 @@
+import { useCallback } from 'react';
 import { useIntl } from 'react-intl';
 
 export type LangFormatFunction = (
   id: string,
-  // TODO: If we pass `undefined | null`, the translation will probably be not correct.
-  // Ensure code using the values parameter do not pass `undefined | null`.
-  values?: Record<string, string | number | Date | undefined | null>,
+  values?: Record<string, string | number | Date>,
```

Comment:
As suggested by the deleted comment, `undefined` and `null` are not valid `values`.